.travis.yml

```diff
@@ -4,13 +4,14 @@ python:
   - 2.6
   - 2.7
   - 3.5
+cache: pip
 matrix:
   allow_failures:
     - python: 3.5
 install:
+  - pip install -U pip coveralls wheel
   - pip install https://github.com/thijstriemstra/python-messaging/archive/master.zip
-  - pip install coveralls
-  - pip install wheel
+  - pip install -r requirements.txt
```

Comment:
This should already be covered by the `pip install -e .` further below as all that does is read the `requirements.txt` file and add it to setup.py's `install_requires`.